.github/workflows/ci.yaml

```diff
@@ -527,27 +523,27 @@ jobs:
             _build/test
             priv/plts
           key: ${{ runner.os }}-${{ env.OTP_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles('mix.lock') }}
-      - name: Generate main openapi.json
-        run: |
-          mix openapi.spec.json --start-app=false --spec TrentoWeb.OpenApi.${{ matrix.version }}.ApiSpec /tmp/specs/main-spec.json
-      - name: Locate generated specs
-        run: mv /tmp/specs .
-      - name: Find difference between OpenAPI specifications
-        run: |
-          docker run -v "$(pwd)/specs:/specs" --rm openapitools/openapi-diff:2.0.1 \
-            /specs/main-spec.json \
-            /specs/current-spec.json \
-            --fail-on-incompatible \
-            --markdown /specs/changes.md \
-            --json /specs/changes.json \
-            --text /specs/changes.txt \
-            --html /specs/changes.html
-      - name: Upload OpenAPI diff report
-        uses: actions/upload-artifact@v4
-        if: failure()
-        with:
-          name: openapi-diff-report-${{ matrix.version }}
-          path: specs/
+      # - name: Generate main openapi.json
```

Comment:
I'm forced to comment this stage out.
We cannot compare specs written in 2 different elixir versions.
Commenting this looks the less invasive way.
I will revert back once this PR is merged

Reply:
Absolutely plus one, no problem for me